homeassistant/components/light/yeelight.py

```diff
@@ -59,6 +64,8 @@
 
 YEELIGHT_FLOW_TRANSITION_SCHEMA = {
     vol.Optional(ATTR_COUNT, default=0): cv.positive_int,
+    vol.Optional(ATTR_ACTION, default=ACTION_RECOVER):
+        vol.Any(ACTION_RECOVER, ACTION_OFF, ACTION_STAY),
```

Comment:
Considering this is optional, update the documentation (and services.yaml) to contain the default value.

Reply:
Updated both.

Reply:
In the code it defaults to ACTION_RECOVER, but the doc states `off` is the default.

Reply:
My bad, I was sleepy, fixed.